pkg/cluster/delete.go

```diff
@@ -126,3 +90,57 @@ func Delete(
 	}
 	return nil
 }
+
+func DeleteOverrideDir(appIf argoapp.ApplicationServiceClient, kubeClient *kubernetes.Clientset, argocdNs string, name string) error {
```

Comment:
A few nits:
- Rename the function to `DeleteOverridesDir()`
- Rename `name` to `clusterName` since it's not clear what thing we're talking about.

Reply:
Renamed both the fields